src/routes/(infoPages)/[topTierPage]/[...serviceGroupPage]/+page.svelte

```diff
@@ -1,7 +1,79 @@
 <script lang="ts">
+  import Accordion, { AccordionItem } from "$lib/components/Accordion";
+  import Card from "$lib/components/Card/Card.svelte";
+  import ContactCard from "$lib/components/ContactCard";
+  import ContentfulRichText from "$lib/components/ContentfulRichText";
+
   export let data;
-  $: ({ pageMetadata } = data);
+
+  $: ({
+    title,
+    subheading,
+    description,
+    heroImage,
+    serviceListName,
+    serviceEntries,
+    serviceGroups,
+    contactInfoCollection,
+    additionalResources,
+  } = data);
 </script>
 
-<h1>{pageMetadata?.title}</h1>
-<p>This is a stub for a Service Group page!</p>
+{#if heroImage}
+  <!-- TODO: Update this to use a proper Hero/Image Component 
+       (pending merge of https://github.com/la-ldaf/ldaf-site/pull/279) -->
+  <img src={heroImage?.imageSource?.url} alt={heroImage.imageSource?.description} />
+{/if}
+<h1>{title}</h1>
+<p class="usa-intro">
+  {subheading}
+</p>
+{#if description}
+  <ContentfulRichText document={description?.json} />
+{/if}
+
+{#if serviceEntries.length > 0 || serviceGroups.length > 0}
+  <h2>{serviceListName}</h2>
+  {#if serviceEntries.length > 0}
+    {#each serviceEntries as item}
+      <Accordion multiselectable>
+        <AccordionItem title={item?.entryTitle} id={item.sys.id}>
+          <ContentfulRichText document={item?.description?.json} />
+        </AccordionItem>
+      </Accordion>
+    {/each}
```

Comment:
Shouldn't the `<Accordion multiselectable>` tag be _outside_ of the `#each` block?

Reply:
yes! good catch